pyproject.toml

```diff
@@ -1,17 +1,11 @@
 [build-system]
-requires = ["setuptools", "wheel", "Cython","numpy"]
-[tool.cibuildwheel]
-# linux uses manylinux2014 as default which uses Centos 7 image
-manylinux-x86_64-image = "manylinux2014"
-before-all = "bash package/scripts/install_prereqs_for_wheels.sh"
```

Comment:
Can we delete the `install_prereqs_for_wheels.sh` script now?

Reply:
Yes, the whole package folder is slated for deletion. Just keeping it until we make a full release to PyPi.